src/shared/common/forms/__tests__/__snapshots__/type-ahead.spec.js.snap

```diff
@@ -440,6 +440,7 @@ exports[`Form type ahead component (select) should render disabled when passed 1
                 id="type-ahead"
```

Comment:
I'm not sure what the Noop is introduced here - this is an Enzyme generated snapshot running in legacy mode via React 18.